Packs/Unshorten_Me/Integrations/unshortenMe/unshortenMe.yml

```
@@ -0,0 +1,36 @@
+category: Utilities
+commonfields:
+  id: unshortenMe
+  version: -1
+configuration:
+- additionalinfo: The token assigned to your unshortenMe profile.
+  display: ' Token'
+  name: token
+  required: true
+  type: 4
```

```suggestion
- additionalinfo: The token assigned to your unshortenMe profile.
  displaypassword: Token
  name: credentials
  required: true
  hiddenusername: true
  type: 9
```
we no longer use type 4, it should be type 9, and i already replaced its usage in the code file

---

Added